backend/routers/todoRouters.js

```diff
@@ -0,0 +1,76 @@
+import { Router } from "express";
+import { getDataBase } from "../db/index.js";
+import { validateValue } from "../helpers.js";
+
+const db = getDataBase();
+const router = new Router();
+
+// Получение всех задач
+router.get("/", async (req, res) => {
+  try {
+    const allTodos = await db.findAsync({});
+    res.send(allTodos);
+  } catch (error) {
+    res.status(500).send({ error: "Failed to receive tasks" });
+  }
+});
+
+// Создание новой задачи
+router.post("/", async (req, res) => {
+  const body = req.body;
+
+  try {
+    await validateValue(body);
+    const task = await db.insertAsync({
+      text: body.text,
+      is_checked: body.is_checked,
+    });
+
+    res.send(task);
+  } catch (error) {
+    res.status(400).send({ error: "Missing required fields" });
+  }
+});
+
+// Удаление задачи
+router.delete("/:id", async (req, res) => {
+  const taskId = req.params.id;
+
+  try {
+    const numRemoved = await db.removeAsync({ _id: taskId });
+
+    if (numRemoved > 0) {
+      return res.send({ message: "Task deleted successfully" });
+    } else {
+      return res.status(404).send({ error: "Task not found" });
+    }
+  } catch (error) {
+    res.status(500).send({ error: "Failed to delete task" });
+  }
+});
+
+// Редактирование задачи
+router.patch("/:id", async (req, res) => {
+  const taskId = req.params.id;
+  const updatedTaskData = req.body;
+
+  const existingTask = await db.findOneAsync({ _id: taskId });
+  if (!existingTask) {
+    return res.status(404).send({ error: "Task not found" });
+  }
+
+  try {
+    await validateValue(updatedTaskData);
+    const { affectedDocuments } = await db.updateAsync(
+      { _id: taskId },
+      { $set: updatedTaskData },
+      { returnUpdatedDocs: true }
+    );
+
+    res.send(affectedDocuments);
```

Comment:
`affectedDocuments` is supposed to be an array of `todo`s, not a single `todo`..?